### PR TITLE
fix: invalid department id type hint

### PR DESCRIPTION
**Pull request by Gaitholabi**
This PR adds 'string' type hint to Email. 

This is necessary because of the type validation pass the serializer does before committing the value through the setter.

The newly added type hint  just tells the serializer to expect a string, however we're enforcing that this string has a valid value in the setter.

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -7,7 +7,6 @@
          convertErrorsToExceptions="true"
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
-         strict="true"
          bootstrap="vendor/autoload.php">
```
Comment (bytestream) [inline phpunit.xml.dist:10]: should be replaced by `beStrictAboutOutputDuringTests`? everything else is enabled by default I think

File: src/Model/Department/Email.php [hunk 2]
```diff
@@ -26,7 +26,7 @@ class Email extends BaseModel
     private $port;
 
     /**
-     * @var int
+     * @var int|string
```
Comment (bytestream) [inline src/Model/Department/Email.php:29]: why is this necessary? the setter enforces `int`

Comment (bytestream) [reply src/Model/Department/Email.php:29]: hmm, says it in the issue:
> This is because of the type validation pass the serializer does before committing the value through the setter.

File: phpunit.xml.dist (hunk 1 above)
Comment (bytestream) [reply phpunit.xml.dist:10]: should probably also be in a separate pr

Comment (Gaitholabi) [reply phpunit.xml.dist:10]: yeah,  seems like enabled by default. guess I reset phpunit to previous state

Comment (bytestream) [reply phpunit.xml.dist:7]: there are a few `beStrict..` options that aren't enabled by default: https://phpunit.readthedocs.io/en/9.5/configuration.html

File: phpunit.xml.dist [hunk 3]
```diff
@@ -4,10 +4,6 @@
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/8.0/phpunit.xsd"
          colors="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
-         strict="true"
```
Comment (bytestream) [inline phpunit.xml.dist:10]: This needs reverting as it's done in the other PR.

**Review by bytestream — APPROVED** [review]

**Review by jshah4517 — APPROVED** [review]
